curio/app/src/components/recorder.js

```diff
@@ -1,11 +1,34 @@
-
-import {Recorder} from 'react-voice-recorder'
+// import {Recorder} from 'react-voice-recorder'
 import 'react-voice-recorder/dist/index.css'
 
-function myRecorder() {
+export function button() {
     return (
-      Recorder
+    <>
+      <form class="w-full max-w-sm">
+        <div class="md:flex md:items-center mb-6">
+          <div class="md:w-1/3">
+            <label class="block text-gray-500 font-bold md:text-right mb-1 md:mb-0 pr-4" for="inline-full-name">
+              Enter Duration <br/> In Seconds
+            </label>
+          </div>
+          <div class="md:w-2/3">
+            <input class="bg-gray-200 appearance-none border-2 border-gray-200 rounded w-full py-2 px-4 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-purple-500" id="inline-full-name" type="number" min={0} max={60} placeholder='1-60'/>
+          </div>
+        </div>
+
+        <div class="md:flex md:items-center">
+          <div class="md:w-1/3"></div>
+          <div class="md:w-2/3">
+            <button class="shadow bg-purple-500 hover:bg-purple-400 hover:text-white focus:shadow-outline focus:outline-none text-gray-500 font-bold py-2 px-4 rounded" type="button">
+              Record
+            </button>
+          </div>
+        </div>
+      </form>
+    </>
     );
   }
+
+
   
-  export default myRecorder;
\ No newline at end of file
+  // export default myRecorder;
```

Comment:
Ditto for the comment!

Also, it is a good practice to leave an empty line at the end of every file.

![image](https://user-images.githubusercontent.com/74055102/187045113-1e63c2cc-e93e-4b99-90f4-c6cbc4c57dae.png)

This shows that your file does not have an empty line at the end.

Reply:
Thank You So Much @Saransh-cpp for your effective suggestions, will definitely take care of all the things I learnt in the future.